addons/account_peppol/wizard/peppol_registration.py

```diff
@@ -0,0 +1,272 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models, modules
+from odoo.exceptions import UserError, ValidationError
+
+from odoo.addons.account_peppol.tools.demo_utils import handle_demo
+
+
+class PeppolRegistration(models.TransientModel):
+    _name = 'peppol.registration'
+    _description = "Peppol Registration"
+
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        required=True,
+        default=lambda self: self.env.company,
+    )
+    account_peppol_contact_email = fields.Char(
```

Comment:
It's a new model so no need to prefix all fields by account_peppol_

Reply:
I thought it would be easier if these fields were named the same across res.company, res.config.settings, and here

Reply:
@aliyatastemirova Remove the prefix :)